Find a functionally equivalent code:com.android.keyguard.KeyguardViewBase.interceptMediaKey:COMMENT
<com.android.keyguard.KeyguardViewBase: boolean interceptMediaKey(KeyEvent)>
Method Modifier: public      private     
Method Modifier: No          Yes         
Method Updated:  Yes         No          
Comment:/**
 * Allows the media keys to work when the keyguard is showing.
 * The media keys should be of no interest to the actual keyguard view(s),
 * so intercepting them here should not be of any harm.
 * @param event The key event
 * @return whether the event was consumed as a media key.
 */

@@ -39,45 +39,47 @@
                         }
                         // Volume buttons should only function for music (local or remote).
                         // TODO: Actually handle MUTE.
-                        mAudioManager.adjustLocalOrRemoteStreamVolume(AudioManager.STREAM_MUSIC, keyCode == KeyEvent.KEYCODE_VOLUME_UP ? AudioManager.ADJUST_RAISE : AudioManager.ADJUST_LOWER);
+                        mAudioManager.adjustSuggestedStreamVolume(keyCode == KeyEvent.KEYCODE_VOLUME_UP ? AudioManager.ADJUST_RAISE : AudioManager.ADJUST_LOWER, /* direction */
+                        AudioManager.STREAM_MUSIC, /* stream */
+                        0);
                         // Don't execute default volume behavior
                         return true;
                     } else {

